apps/consent-ui/src/components/Header/LanguageToggle.tsx

```diff
@@ -20,29 +20,29 @@
 'use client';
 import { usePathname } from 'next/navigation';
 import { ReactNode } from 'react';
+import clsx from 'clsx';
 
+import styles from '@/components/Button/Button.module.scss';
 import { ValidLanguage } from '@/i18n';
-import LinkButton from '@/components/Button/LinkButton';
 
-const getPathWithNewLocale = (locale: ValidLanguage, pathname?: string) => {
-	if (!pathname) return '/';
-	const segments = pathname.split('/');
-	segments[1] = locale;
-	return segments.join('/');
-};
+import LocalizedLink from '../Link/LocalizedLink';
+import { findLinkNameByPath, getUnselectedLang } from '../Link/utils';
```

Comment:
just to to keep things consistent

```suggestion
import LocalizedLink from '@/components/Link/LocalizedLink';
import { findLinkNameByPath, getUnselectedLang } from '@/components/Link/utils';
```

Reply:
oh yeah derp